### DISCUSSION TITLE
Requesting help showing ilvl under my character name (old post for reference)

### DISCUSSION THREAD

**Discussion by daviddavidsonagain** [SavedInstances/SavedInstances#893] [Q&A]
A year ago, [a thread popped up that showcased how to expose your character's ilvl under their name](https://github.com/SavedInstances/SavedInstances/issues/750) (instead of having to hover your mouse cursor over the character's name).

I'm not knowledgeable when it comes to coding (ironic considering the site I'm on, I know), but I was able to follow the instructions laid out in that thread and it worked flawlessly. However, now that TWW has launched, those instructions no longer work, as the core.lua file has, of course, been updated and things have moved around.

Is there anyone who can look at that thread I've linked above and figure out how to accomplish the task once again? I would really, really appreciate it. Thank you for reading and potentially helping. <3

**Comment by LiangYuxuan** [comment]
Now the line is

https://github.com/SavedInstances/SavedInstances/blob/c44dc082e347e4884b9cbd25c8aff76055819a59/SavedInstances/Core/Core.lua#L4459

**Reply by daviddavidsonagain** [reply]
Thank you so much for taking the time to respond. I actually found the new line earlier, but the code that was made available in last year's thread no longer works (assuming I'm copying/pasting it correctly). Do you happen to know what new string I would need to replace line 4459 with?

I'll show you what I've [copied and pasted into line 4459](https://i.imgur.com/ijTq7Oa.png) for reference. This is actually line 4449 to 4471.

**Comment by daviddavidsonagain** [comment]
Would anyone else be available to take a crack at this? I can pay in compliments and gratefulness! 😄
